server/src/main/java/org/candlepin/model/OwnerProductCurator.java

```diff
@@ -568,41 +569,51 @@ public void updateOwnerProductReferences(Owner owner, Map<String, String> produc
 
         Session session = this.currentSession();
 
-        Map<String, Object> criteria = new HashMap<>();
+        Map<String, Object> ownerProductCriteria = new HashMap<>();
```

Comment:
This change was not related to performance. This is the place where manifest import fails with too many params. I did some renaming to better see what is happening with all those reassignments. I can revert those to to prevent conflicts but partitioning needs to stay otherwise import will fail.